roles/common/tasks/main.yml

```diff
@@ -20,6 +20,26 @@
     retries: 5
   when: ansible_distribution == "Ubuntu"
 
+- name: set selinux to permissive on redhat
+  selinux:
+    policy: targeted
+    state: permissive
+  when: ansible_distribution == "RedHat"
+  tags: selinux
+
+- name: Register with RHN
+  redhat_subscription:
+    state: present
+    username: "{{ redhat.user }}"
+    password: "{{ redhat.pass }}"
+  when: ansible_distribution == "RedHat"
+  tags: rhn
+
```

Comment:
odd, not sure how this is working without specifying which pool to attach to. Anyways, @twaldrop PR on paul_fixes replaces this code.